one-pager-maker/src/redux/user/userSlice.ts

```diff
@@ -0,0 +1,23 @@
+import {createSlice} from '@reduxjs/toolkit'
+import {InitialUserState} from './userType.ts'
+
+const initialState: InitialUserState = {
+    displayName: "",
+    user: null,
+}
+
+export const userSlice = createSlice({
```

Comment:
You can get currentUser data by `auth.currentUser` so this store may be unnecessary.

Reply:
Thank you.  
I prepared a slice to detect the state of the login, but is it unnecessary?  
Since I'm not sure how to delete, would it be okay to remove when refactoring eventually?

Reply:
Yes yes that's fine
thank you